### PR TITLE
added loggings to find root of inttests error

### PR DESCRIPTION
**Pull request by Akyiaa**
Signed-off-by: Fiona Ampofo <64271621+Akyiaa@users.noreply.github.com>

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

File: galasa-maven-plugin/pom.xml [hunk 1]
```diff
@@ -154,6 +153,13 @@
 			<artifactId>assertj-core</artifactId>
 			<version>3.25.3</version>
 		</dependency>
+		
+		<dependency>
```
Comment (techcobweb) [inline galasa-maven-plugin/pom.xml:157]: Should be 
```
<dependency>
    <groupId>commons-logging</groupId>
    <artifactId>commons-logging</artifactId>
    <version>1.3.0</version>
</dependency>
```

**Review by techcobweb — DISMISSED** [review]
Subject to the suggested change being applied.

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Review by techcobweb — APPROVED** [review]

**Pull request merged**